docs-2.0/6.monitor-and-metrics/1.query-performance-metrics.md

```diff
@@ -103,4 +105,116 @@ curl -G "http://<ip>:<port>/stats?stats=<metric_name_list> [&format=json]"
     num_heartbeats.sum.60=40
     num_heartbeats.sum.600=394
     num_heartbeats.sum.3600=2364
+    ...
     ```
+
+## Metric description
+
+### Graph
+
+| Parameter                                           | Description                                                         |
+| ---------------------------------------------- | ------------------------------------------------------------ |
+| `num_active_queries`                             | The number of queries currently being executed.              |
+| `num_active_sessions`                            | The number of currently active sessions.                     |
+| `num_aggregate_executors`                        | The number of executions for the Aggregation operator.       |
+| `num_auth_failed_sessions_bad_username_password` | The number of sessions where authentication failed due to incorrect username and password. |
+| `num_auth_failed_sessions_out_of_max_allowed` |  The number of session that failed to authenticate logins because the parameter `FLAG_OUT_OF_MAX_ALLOWED_CONNECTIONS` was exceeded.|
```

Comment:
```suggestion
| `num_auth_failed_sessions_out_of_max_allowed` |  The number of sessions that failed to authenticate logins because the value of the parameter `FLAG_OUT_OF_MAX_ALLOWED_CONNECTIONS` was exceeded.|
```